bio/macs2/callpeak/environment.yaml

```
@@ -0,0 +1,5 @@
+channels:
+  - bioconda
+  - defaults
+dependencies:
+  - macs2 ==2.2.7.1
```

The continuous integration error comes from conda being unable to resolve this environment. Maybe conda chokes on the long version number, which is one entry longer than semantic versioning allows, so maybe try (it's just a hunch, but who knows...;):
```suggestion
  - macs2 ==2.2.7
```